thrift.js

```diff
@@ -360,9 +362,21 @@ Thrift.prototype.link = function link() {
 
     this.exception.link(this);
 
+    // As part of the linking process, we also release aliased
+    // refs to idls, asts, etc.
+    if (this.releaseSources) {
+        this.releaseResources();
+    }
+
     return this;
 };
 
+Thrift.prototype.releaseResources = function release() {
```

Comment:
add documentation on what this does and when should this be invoked?

Reply:
This is a private method, inlined it actually